partner-onboarder/install.sh

```
@@ -57,7 +57,9 @@ function installing_onboarder() {
           echo "S3 access key not provided; EXITING;"
           exit 1;
         fi
-        read -p "Please provide S3 secret key: " s3_secret_key
+        echo "Please provide S3 secret key"
+        read -s -p "" s3_secret_key
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)